template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/Microsoft.DotNet.Common.ProjectTemplates.6.0.csproj

```diff
@@ -7,6 +7,12 @@
         <OutputPath>$(TemplatesDir)</OutputPath>
         <EnableDefaultItems>False</EnableDefaultItems>
         <IsPackable>true</IsPackable>
+        <!-- ship for preview, ship for rc, ship for rtm, do not ship for servicing -->
```

Comment:
Maybe we should mention why 6.0 is different than others so that devs will know what to do when .NET 7 comes around. 
"Ship with previews since .NET 6 isn't yet released."